site/app/views/grading/ElectronicGraderView.php

```diff
@@ -667,17 +667,14 @@ function display_files($files, &$count, $indent, &$return) {
             <input type="hidden" name="g_id" value="{$gradeable->getId()}" />
             <input type="hidden" name="u_id" value="{$user->getId()}" />
             <input type="hidden" name="individual" value="{$individual}" />
+            <input type="hidden" name="graded_version" value="{$gradeable->getActiveVersion()}" />
```

Comment:
The graded_version is only the activeVersion the first time you submit, else it should be the saved graded_version number that's stored in the DB.